From: SeongJae Park <sj@kernel.org>
Date: Sun, 22 Jun 2025 12:24:47 -0700
Subject: [PATCH] mm/damon: change definition of active_mem_bp

Change from active to inactive mmeory ratio, to active to (active +
inactvie) memory ratio.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 include/linux/damon.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/damon.h b/include/linux/damon.h
index 7c2c58e9cba2..4b4d3669ba42 100644
--- a/include/linux/damon.h
+++ b/include/linux/damon.h
@@ -177,7 +177,7 @@ enum damos_action {
  * @DAMOS_QUOTA_NODE_MEM_FREE_BP:	MemFree ratio of a node.
  * @DAMOS_QUOTA_NODE_MEMCG_USED_BP:	MemUsed ratio of a node for a cgroup.
  * @DAMOS_QUOTA_NODE_MEMCG_FREE_BP:	MemFree ratio of a node for a cgroup.
- * @DAMOS_QUOTA_ACTIVE_MEM_BP:		Active to inactive memory ratio.
+ * @DAMOS_QUOTA_ACTIVE_MEM_BP:		Active to [in]active memory ratio.
  * @NR_DAMOS_QUOTA_GOAL_METRICS:	Number of DAMOS quota goal metrics.
  *
  * Metrics equal to larger than @NR_DAMOS_QUOTA_GOAL_METRICS are unsupported.
-- 
2.39.5

